src/parser.cpp

```diff
@@ -325,7 +335,7 @@ void Parser::ParserImpl::loadModel(const ModelPtr &model, const std::string &inp
             }
         } else if (childNode->isType("connection")) {
             connectionNodes.push_back(childNode);
-        } else if (childNode->isType("text")) {
+        } else if (childNode->isType(NULL_NS, "text")) {
```

Comment:
How do you tell the difference between an actual text node, and an element named 'text' in the null namespace?

Reply:
Ok, I clearly misunderstood @jonc125's point. I think I now understand the problem and something like what @nickerso suggests is probably the way to go indeed. In fact, we should do something similar for comment nodes.